bin/lib/common.sh

```diff
@@ -47,7 +47,7 @@ function yell () {
 }
 
 function die() {
-    recho "$(basename $0): $*" >&2
+    recho "$(basename $0): ERROR: $*" >&2
     exit 111
```

Comment:
not to be pedantic... but did you check the use of this in the MANY other places its used? is "error:" really necessary?

Reply:
well,  we do an exit afterwards and so suggests the function name. So clearly it is considered an error (except of course the exit is ignored on some places where we do `: good-condition || $(die ....)` that this is clearly still semantically an error.
I actulaly also add a WARN to yell but we seem to use yell also for non-warning cases so i dropped that.  The main reason i added that is that i wanted to be able to grep for these messages in the voluminous log-fiels .  I guess there is a way to also grep for the color red but i think these seemed cleaner and works even on b/w terminals ...